### PR TITLE
[FIX] check seeds dtype

### LINKED ISSUE #2034: Using the tutorial on Euler method on my data
## Description
Hi I am getting a weird error I cannot debug when I am running the streamline generator. What I m doing is simply using your tutorial on Eu method but with my personal data. I am using Dipy 1.0.0 and Python3.6 .

this is the full list of the error:
```python
Traceback (most recent call last):
  File "tracking_introduction_eudx.py", line 167, in <module>
    streamlines = Streamlines(streamlines_generator)
  File "/home/alecrimi/.local/lib/python3.6/site-packages/nibabel/streamlines/array_sequence.py", line 135, in __init__
    self.extend(iterable)
  File "/home/alecrimi/.local/lib/python3.6/site-packages/nibabel/streamlines/array_sequence.py", line 315, in extend
    for e in elements:
  File "/home/alecrimi/.local/lib/python3.6/site-packages/dipy/tracking/utils.py", line 880, in transform_tracking_output
    for sl in streamlines:
  File "/home/alecrimi/.local/lib/python3.6/site-packages/dipy/tracking/local_tracking.py", line 120, in _generate_streamlines
    for s in self.seeds:
TypeError: 'int' object is not iterable
/home/alecrimi/.local/lib/python3.6/site-packages/matplotlib/backends/backend_gtk3.py:197: Warning: Source ID 8 was not found when attempting to remove it
  GLib.source_remove(self._idle_draw_id)
```

this is how I changed the code
```python 
interactive = False
import nibabel as nib
from dipy.core.gradients import gradient_table
from dipy.data import get_fnames
from dipy.io.gradients import read_bvals_bvecs
from dipy.io.image import load_nifti#, load_nifti_data

#hardi_fname, hardi_bval_fname, hardi_bvec_fname = get_fnames('stanford_hardi')
#label_fname = get_fnames('stanford_labels')
#data, affine, hardi_img = load_nifti(hardi_fname, return_img=True)
#labels = load_nifti_data(label_fname)
#bvals, bvecs = read_bvals_bvecs(hardi_bval_fname, hardi_bvec_fname)
#gtab = gradient_table(bvals, bvecs)

fimg = "DTICAP_bet.nii.gz"
img = nib.load(fimg)
data = img.get_data()
fbval = "DTICAP.bval"
fbvec = "DTICAP.bvec"
affine = img.affine

bvals, bvecs = read_bvals_bvecs(fbval, fbvec)
gtab = gradient_table(bvals, bvecs)

#white_matter = (labels == 1) | (labels == 2)

from dipy.reconst.csdeconv import auto_response
from dipy.reconst.shm import CsaOdfModel
from dipy.data import default_sphere
from dipy.direction import peaks_from_model

response, ratio = auto_response(gtab, data, roi_radius=10, fa_thr=0.7)
csa_model = CsaOdfModel(gtab, sh_order=6)
csa_peaks = peaks_from_model(csa_model, data, default_sphere,
                             relative_peak_threshold=.5,
                             min_separation_angle=25,
                             )
                             #mask=white_matter)

 
from dipy.tracking.stopping_criterion import ThresholdStoppingCriterion

stopping_criterion = ThresholdStoppingCriterion(csa_peaks.gfa, .25)
 

import matplotlib.pyplot as plt

sli = csa_peaks.gfa.shape[2] // 2
plt.figure('GFA')
plt.subplot(1, 2, 1).set_axis_off()
plt.imshow(csa_peaks.gfa[:, :, sli].T, cmap='gray', origin='lower')

plt.subplot(1, 2, 2).set_axis_off()
plt.imshow((csa_peaks.gfa[:, :, sli] > 0.25).T, cmap='gray', origin='lower')

plt.savefig('gfa_tracking_mask.png')
 
  
from dipy.tracking import utils

#seed_mask = (labels == 2)
#seeds = utils.seeds_from_mask(seed_mask, affine, density=[2, 2, 2])

from dipy.tracking.local_tracking import LocalTracking
from dipy.tracking.streamline import Streamlines

#Initialization of LocalTracking. The computation happens in the next step.
streamlines_generator = LocalTracking(csa_peaks, stopping_criterion, seeds=100000,
                                      affine=affine, step_size=.5)
#Generate streamlines object
streamlines = Streamlines(streamlines_generator)
 
from dipy.io.stateful_tractogram import Space, StatefulTractogram
from dipy.io.streamline import save_trk

sft = StatefulTractogram(streamlines, img, Space.RASMM)
save_trk(sft, "tractogram_EuDX.trk", streamlines)
```

### PR DESCRIPTION
**Pull request by skoudoro**
A small PR to fix #2034. I just check the `seed` type to avoid an error later with a much more complex message.

### CODE REVIEW

File: dipy/tracking/local_tracking.py [hunk 1]
```diff
@@ -79,6 +80,9 @@ def __init__(self, direction_getter, stopping_criterion, seeds, affine,
             raise ValueError("step_size must be greater than 0.")
         if maxlen < 1:
             raise ValueError("maxlen must be greater than 0.")
+        if not isinstance(seeds, Iterable):
+            raise ValueError("seeds should be a list of 3D points. Please"
```
Comment (arokem) [inline dipy/tracking/local_tracking.py]: This seems a bit confusing to me. The first sentence says the input should be a list and then the second sentence says it should be an array. I think either is fine? Could you please clarify that?

**Comment by arokem** [comment]
Looks like conda on OSX is having a bad morning. Have you seen that one before?

File: dipy/tracking/local_tracking.py (hunk 1 above)
Comment (skoudoro) [reply dipy/tracking/local_tracking.py]: Ok :+1: , will do

**Comment by skoudoro** [comment]
Yes, I saw that on FURY. Sometimes, it is just a server problem and we have to be patient. Sometimes, the lastest miniconda version is just not working correctly so we have to downgrade the miniconda version and not get the latest one.

I will wait a bit to commit my change and let's see.

**Comment by arokem** [comment]
YLGTM. Since this is small, and has already gone through one round of review, I will go ahead and merge this now.

**Pull request merged**